src/components/Pages/PeoplePage/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,75 @@
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import { useLocation, useSearchParams } from 'react-router-dom';
+import { useEffect, useMemo, useState } from 'react';
+import { Person } from '../../../../types';
+import { SearchParamsTypes } from '../../../../types/SearchParamsTypes';
+import { TableHeaderItem, PersonBlock } from './index';
+
+import { filterPeople, sortsPeople } from '../../../../utils';
+
+const {
+  Query, Centuries, Sex, Order, Sort,
+} = SearchParamsTypes;
+
+type Props = {
+  peopleList: Person[],
+};
+
+export const PeopleTable: React.FC<Props> = ({ peopleList }) => {
+  const TableHeaderList = ['Name', 'Sex', 'Born', 'Died'];
```

Comment:
```suggestion
  const tableHeaderList = ['Name', 'Sex', 'Born', 'Died'];
```
It's not a component, so first letter shouldn't be capitalized